docs/source/handlers.rst

```diff
@@ -107,8 +107,11 @@ handler might accept ``application/x-www-form-urlencoded`` and
 
 If ``accepts`` is defined two additional static methods should be defined:
 
-* ``dumps``: Turn structured Python data from the ``data`` key in a
-  test into a string or byte stream.
+* ``dumps``: Turn structured Python data from the ``data`` key in a test into a
+  string or byte stream. The optional ``test`` param allows you to access the
+  current test case which may help with manipulations for custom content
+  handlers, e.g. multipart/form-data needs to add a `boundary` to the
```

Comment:
Sorry, I meant to say 'content-type' and then it might have been more clear that I meant ``multipart/form-data`` (because it's a literal).